TDP/tdp-44.md

```diff
@@ -0,0 +1,69 @@
+---
+tdp: 44
+title: mStableDAO Restructure
+status: WIP
+author: mZeroNine (@mzero2nine)
+discussions-to: https://forum.mstable.org
+created: 2022-06-20
+---
+
+## Simple Summary
+
+It is proposed to restructure the mStableDAO in order to increase its efficiency as well as overall ability to scale in the future without further re-adjustments down the line in order to prepare for the release of mStable v2. 
+
+Since the restructuring process as a whole is extensive, it is also proposed to split the mStableDAO restructuring into multiple TDPs in order to maintain explicitness in the voting outcome as well as ensuring a digestible format for Meta Governors. The entire process will contain 4 TDPs in total and is planned to conclude by the 3rd week of August 2022 the latest.
+
+TDP 44: mStableDAO Restructure is concerned with the overall consensus on
+
+- Whether or not Meta Governors wish the mStableDAO to be restructured as described below
+- The format of having subsequent TDPs that will resolve more granular details of the restructuring process as a whole
+
+## Abstract
+
+This first TDP is targeted at the overall proposition to begin the restructuring of the mStableDAO. 
+
+The key actions proposed include:
+
+- Introducing a general universal contributor title which would involve removing current individual rules and holding a full election process (part of the next TDP) for each of the available seats inside of the respective DAO. Both DAOs will operate under a 4/6 multisig.
+- Merging the Asset Management subDAO with the TreasuryDAO and absorb primary funding functions of the Funding subDAO back into the TreasuryDAO.
+- Creation of a new “Product/Builder subDAO” to separate the ProtocolDAO from the full-time contributor's group that is responsible for the continuous roll-out and operations of the mStable product line. The creation of this subDAO will ensure the long-term success of the protocol while preventing any centralized points of failure or sensitivity to external threats

```

Comment:
`Product/Builder subDAO` Is that the final name?

Reply:
I am fine with this, but since it is the proposal, it should imo specify the name :D